utils/src/main/scala/za/co/absa/enceladus/utils/time/TimeZoneNormalizer.scala

```diff
@@ -38,6 +40,11 @@ object TimeZoneNormalizer {
     }
   }
 
+  /**
+    * This is just an empty function suitable to be called to ensure the object is created and therefore JVM time zone normalized
+    */
+  def activate(): Unit = {}
```

Comment:
Why not call `normalizeJVMTimeZone()` everywhere you call now `activate()`?

Reply:
The `normalizeJVMTimeZone()` is enough to execute only once. And at the same time it's a third party outside logic, which cost I don't know - therefore I try to avoid unnecessary multiple calls.

Reply:
I'd like to get CQC opinion on this.

Reply:
That's really a cool idea. It's fast to explain the background of this **optional** change. And I can than add my reasoning for the chosen solution. 👍 
And this is not really a hot issue, the original bug was fixed in the other PR.